recipes/geo-espresso/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "geo-espresso" %}
+{% set version = "0.2.2.dev1" %}
```

Comment:
We generally want to package release builds for conda-forge. Does this package have any regular releases?

Reply:
Thanks for the comment. We haven't had a release yet, but maybe that's due to us being too conservative about versioning. The API is relatively stable already, but currently the user base is small so I'm not entirely sure we are ready in terms of having as fewer bugs as possible.

If it's really necessary, I can make a release for going to conda-forge. Otherwise I think it should be fine and I'm pretty sure the development is ongoing (not going to be abandoned or something)..

Reply:
Hi, we've released v0.3.0 on PyPI and I'll update the version number and hash after [the conversation below](https://github.com/conda-forge/staged-recipes/pull/22613#issuecomment-1519484088) has been reviewed. Thanks and cheers!